server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -117,7 +117,42 @@ func (s *BuildBuddyServer) CreateUser(ctx context.Context, req *uspb.CreateUserR
 }
 
 func (s *BuildBuddyServer) CreateGroup(ctx context.Context, req *grpb.CreateGroupRequest) (*grpb.CreateGroupResponse, error) {
-	return nil, status.UnimplementedError("Not Implemented")
+	auth := s.env.GetAuthenticator()
+	userDB := s.env.GetUserDB()
+	if auth == nil || userDB == nil {
+		return nil, status.UnimplementedError("Not Implemented")
+	}
+	jwtUser := &tables.User{}
+	if err := auth.FillUser(ctx, jwtUser); err != nil {
+		return nil, err
+	}
+
+	groupName := strings.TrimSpace(req.Name)
+	if len(groupName) == 0 {
+		return nil, status.InvalidArgumentError("Group name cannot be empty")
+	}
+
+	var groupOwnedDomain string = ""
```

Comment:
this is fine, but if you want to be more concise you can do:
groupOwnedDomain := ""

Typically the "var x type" idiom is used when you  have some conditional assignment happening inside a scope or similar.

Reply:
Done